stack/docker-compose.yml

```
@@ -202,5 +207,19 @@ services:
+  # docker-compose --profile=aenderungsservice up
+  # oder e.g.:
+  # docker-compose --profile=backend --profile=frontend --profile=online --profile=aenderungsservice --profile=eai up
+  aenderungsservice:
+    profiles:
+      - aenderungsservice
+    image: ghcr.io/it-at-m/ehrenamt-justiz/ehrenamt-justiz-aenderungsservice:latest
+    ports:
+      - "8086:8086"
+    healthcheck:
+      <<: *healthcheck
+      test: [ "CMD-SHELL", "curl http://localhost:8086/actuator/health/liveness" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)